lms/djangoapps/grades/tests/test_course_grade_factory.py

```diff
@@ -106,12 +106,18 @@ def _assert_section_order(course_grade):
         with self.assertNumQueries(2):
             _assert_read(expected_pass=True, expected_percent=0.5)  # NOT updated to grade of .25
 
-        with self.assertNumQueries(15), mock_get_score(2, 2):
+        with self.assertNumQueries(12), mock_get_score(2, 2):
             grade_factory.update(self.request.user, self.course, force_update_subsections=True)
 
         with self.assertNumQueries(2):
             _assert_read(expected_pass=True, expected_percent=1.0)  # updated to grade of 1.0
 
+        with self.assertNumQueries(12), mock_get_score(0, 0):  # the subsection now is worth zero
+            grade_factory.update(self.request.user, self.course, force_update_subsections=True)
+
+        with self.assertNumQueries(2):
+            _assert_read(expected_pass=False, expected_percent=0.0)  # updated to grade of 1.0
```

Comment:
Need to update comment to say `updated to grade of 0.0`.

Reply:
Have you verified that this test fails on master?

Reply:
It does fail on master, and I'll update that!